GOVERNANCE.md

```diff
@@ -22,30 +18,222 @@ New maintainers must be nominated by an existing maintainer and must be elected
+or via the [Pinniped mailing list](mailto:project-pinniped@googlegroups.com).
+
+Author(s) of proposals for major changes will give a time period of no less than five (5) working days
+for comment and remain cognizant of popular observed world holidays.
+
+If you don't already have a maintainer to sponsor your proposal, then reach out via Slack or the mailing list
+with a description of the problem statement. If one or more of the maintainers agrees that the problem
+statement is within the scope of the project (see [SCOPE.md](SCOPE.md)) and is appropriate to be addressed by a proposal,
+then a maintainer will be assigned as your proposal's sponsor. The sponsor can provide you with support during
+the drafting of the proposal, including sharing additional project and roadmap context as it relates to your
+problem statement, answering questions, giving feedback, etc.
+
+### Proposal Template
+The below template is an example `README.md` for a new proposal.
+Other than the high-level details at the top of the template (title, authors, status, sponsor, and approval_date)
+and the disclaimer at the top, please use whichever sections make the most sense for your proposal.
+
+```md
+---
+title: "The Name of My Proposal"
+authors: [ "@margocrawf", "@enj" ]
+status: "draft"
+sponsor: [ "@cfryanr" ]
+approval_date: ""
 ---
-# Updating Governance
-All substantive changes in Governance require a supermajority agreement by all maintainers.
\ No newline at end of file
+
+*Disclaimer*: Proposals are point-in-time designs and decisions.
+Once approved and implemented, they become historical documents.
+If you are reading an old proposal, please be aware that the
+features described herein might have continued to evolve since.
+
+# <Proposal Title>
+
+## Problem Statement
+This is a short summary of the problem that exists, why it needs to be
+solved: what specific needs are being met. Compelling problem statements
+include concrete examples and use cases (even if only by reference).
+How exactly the proposal would meet those needs should be located in the
+"Proposal" section, not this one. The goal of this section is to help
+readers quickly empathize with the target users' current experience to
+motivate the proposed change.
+
+### How Pinniped Works Today (as of version vX.X.X)
+How Pinniped works today in the context of the problem statement.
+This will typically detail how Pinniped falls short of supporting
+the desired use case(s).
+
+## Terminology / Concepts
+Define any terms or concepts that are used throughout this proposal.
+
+## Proposal
+The primary content of the proposal. Subsections will explain how the
+problem(s) will be addressed.
+
+### Goals and Non-goals
+A short list of what the goals of this proposal are and are not.
+
+### Specification / How it Solves the Use Cases
+Detailed explanation of the proposal's design. This will typically
+also detail how the specification supports the desired use cases.
+
+#### API Changes
```

Comment:
Maybe also add "what will defaults be for the new APIs"

Reply:
Added.